colearn_examples/ml_interface/mli_fraud.py

```diff
@@ -111,7 +111,7 @@ def mli_get_current_model(self) -> ColearnModel:
         """
 
         return ColearnModel(
-            model_format=ModelFormat(2),
+            model_format=ModelFormat(ModelFormat.ONNX),
```

Comment:
I think this is equivalent to `model_format=ModelFormat.ONNX` which is what I meant by my previous comment! Minor though.